### PR TITLE
Instructor: help page: improve add/edit course-related content #8635

### PR DESCRIPTION
**Pull request by craaaa**
Part of #8635

- Rename section to 'Courses'
- Edit content
- Move content into collapsibles
- Move `Sections` and `Archiving` content to 'Courses'

### CODE REVIEW

**Comment by Haozhe321** [comment]
@craaaa I will take a look at this and the other PR by tmr!

**Comment by craaaa** [comment]
Added most of the course-related information to the same section, called `Courses`. Live demo can be viewed [here](https://teammates-cara.appspot.com/instructorHelp.jsp#editCourse).

File: src/main/webapp/partials/instructorHelpCourses.jsp [hunk 1]
```diff
@@ -0,0 +1,730 @@
...
+    <div class="panel panel-primary" id="editCourseSetAccessLevel">
+      <div class="panel-heading">
+        <h3 class="panel-title">
+          <a data-toggle="collapse" href="#editCourseSetAccessLevelBody">How do I set an instructor's access level?</a>
+        </h3>
+      </div>
+      <div id="editCourseSetAccessLevelBody" class="panel-collapse collapse">
+        <div class="panel-body">
+          <p>
+            When <a data-toggle="collapse" data-target="#editCourseAddInstructorBody" href="#editCourseAddInstructor">adding an instructor</a> or <a data-toggle="collapse" data-target="#editCourseEditInstructorBody" href="#editCourseEditInstructor"> editing an instructor's information</a>, you can set the instructor's access level.
```
Comment (Haozhe321) [inline src/main/webapp/partials/instructorHelpCourses.jsp]: The 2 links will close the earlier sub-sections if they are already open.

File: src/main/webapp/partials/instructorHelpCourses.jsp [hunk 2]
```diff
@@ -0,0 +1,730 @@
...
+    <div class="panel panel-primary" id="archivingUnarchiveCourses">
+      <div class="panel-heading">
+        <h3 class="panel-title">
+          <a data-toggle="collapse" href="#archivingUnarchiveCoursesBody">How do I unarchive an archived course?</a>
+        </h3>
+      </div>
+      <div id="archivingUnarchiveCoursesBody" class="panel-collapse collapse">
+        <div class="panel-body">
+          <p>
+            To unarchive a course, first <a data-toggle="collapse" data-target="#archivingViewArchivedCoursesBody" href="#archivingViewArchivedCourses">view the course</a> that you would like to unarchive in the <b>Courses</b> page.<br>
```
Comment (Haozhe321) [inline src/main/webapp/partials/instructorHelpCourses.jsp]: Same issue with collapsing the sub-section.

**Comment by Haozhe321** [comment]
Also, I think it's worth looking into the colour for the collapsible bar. Right now it looks the same as the UI of our app(i.e. questions, instructor). I wonder if gray would look good for the bar. 
image

**Pull request closed without merging**